lib/blockchain.go

```diff
@@ -2393,14 +2415,16 @@ func (bc *Blockchain) CreatePrivateMessageTxn(
 		PublicKey: senderPublicKey,
 		TxnMeta: &PrivateMessageMetadata{
 			RecipientPublicKey: recipientPublicKey,
-			EncryptedText:      encryptedText,
+			EncryptedText:      encryptedMessageBytes,
 			TimestampNanos:     tstampNanos,
 		},
 
 		// We wait to compute the signature until we've added all the
 		// inputs and change.
 	}
 
+	txn.ExtraData = messageExtraData
```

Comment:
Sorry I missed this last review. Please move this into the struct initalization: `ExtraData: messageExtraData `

Reply:
Oh my bad, missed this too.